ITR/interfaces.py

```diff
@@ -594,7 +734,7 @@ def _fixup_historic_data(self, historic_data, production_metric, emissions_metri
         return model_historic_data
 
     def _get_base_realization_from_historic(self, realized_values: List[PintModel], units, base_year=None):
-        valid_realizations = [rv for rv in realized_values if np.isfinite(rv.value)]
+        valid_realizations = [rv for rv in realized_values if not unp.isnan(rv.value)]
```

Comment:
doesn't `np.isfinite` do exactly the same as `not unp.isnan` in our context?
why do we choose to replace it in several places?

Reply:
No, because np.isfinite does not work with ufloats (uncertainties).